### ISSUE TITLE
Start adb network debugging without root or root

### ISSUE THREAD

**Issue by ranrui-y** [rustdesk/rustdesk#4002]
### Description

Screen touch stabilization through accessibility cannot be used Sometimes strikes.

This is the way to add more startup to the software. Perpetual accessibility does not control the equipment very well.

Start adb network debugging through root or authorize shizuku through PC, and shizuku will distribute temporary adb.

### Impact

Users will benefit.

### Additional Context

With accessibility, it is only single-touch and unstable.

**Comment by ranrui-y** [comment]
通过无障碍实现，只能单点触控且不稳定。
